src/modules/editTask.js

```diff
@@ -0,0 +1,27 @@
+import { TasksClass } from './tasksClass.js';
+
+// edit new task by click event
+export const editTask = () => {
+  //  get the elements
+  const editElement = document.querySelectorAll('.task-text');
+  const editTask = new TasksClass();
+  // get index and text content by iterating
+  editElement.forEach((el, elIndex) => {
+    // functions for editing
+    const editContent = () => {
+      el.setAttribute('contenteditable', 'true');
+      el.style.backgroundColor = '#ffffcb';
+    };
+    const taskUpdate1 = () => {
+      const taskItemInput = el.innerText;
+      editTask.taskEditor(elIndex, taskItemInput);
+    };
+    const taskUpdate2 = () => el.setAttribute('contenteditable', 'true');
+    // events
+    el.addEventListener('dblclick', editContent, false);
+    el.addEventListener('input', taskUpdate1, false);
+    el.addEventListener('keydown', taskUpdate2, false);
+  });
```

Comment:
- [ ] Great job on implementing the edit and remove task functionality however there is a slight problem with it. Currently both functions don't work until we refresh the page. See the screen cast for more details:


https://github.com/ometman/To-Do-List-2/assets/44798044/a07f640c-c168-43e8-8519-0efdf301541a



Please fix this problem as this is not good for user experience also it's not very obvious how that the app needs to be refreshed to make it work.

Reply:
Thanks. Fixed.

Reply:
- Hi, according to the previous reviewers' requested changes, the changes have not been well resolved.
I still get to refresh before the delete icon shows,  and also for the edit function. please, kindly improve the user experience so that when users click on the three-dotted vertical lines icon, it should show the delete icon and also trigger the edit function. instead of having to keep refreshing your page before an action can be made. please, check his referred screencast again to fix the issue.